src/Features/Admin/Categories/index.js

```diff
@@ -21,7 +22,24 @@ function CategoryList() {
     row;
   };
 
-  const header = ['No', 'Category', 'Edit'];
+  const [categories, setCategories] = useState([]);
+
+  const getCategoriesResponseFromEpic = (response) => {
+    setCategories(response);
+  };
+
+  const [header, setHeader] = useState([]);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchCategories(getCategoriesResponseFromEpic));
```

Comment:
Why you are passing the function from component is it a good practise you learn from somewhere or is it a try ??